crates/web-plugins/didcomm-messaging/shared/src/retry.rs

```diff
@@ -0,0 +1,75 @@
+use std::time::Duration;
```

Comment:
Wow you already implemented the retry_logic here :+1: 
That makes less work for me on this [ticket](https://github.com/adorsys/didcomm-mediator-rs/issues/259)

Reply:
:smile: great